base_local_planner/src/latched_stop_rotate_controller.cpp

```diff
@@ -85,9 +97,14 @@ bool LatchedStopRotateController::isGoalReached(LocalPlannerUtil* planner_util,
 
   base_local_planner::LocalPlannerLimits limits = planner_util->getCurrentLimits();
 
+  const double goal_dist = base_local_planner::getGoalPositionDistance(global_pose, goal_x, goal_y);
+  const bool reached_outer_goal = goal_dist <= xy_goal_tolerance;
+  const bool reached_inner_goal = goal_dist <= limits.inner_xy_goal_tolerance;
```

Comment:
S: maybe take same way as xy_tolerance?  maybe both from limits?

Reply:
They're both from limits, xy_goal_tolerance is simply stored a little bit above in the code. There's no need to create an extra variable, but sure -- I'll keep the same style.

Fixed.